types/office-js-preview/index.d.ts

```diff
@@ -3654,6 +3666,35 @@ declare namespace Office {
          */
         forMSGraphAccess?: boolean;
     }
+    /**
+     * Object with user information which can be passed to msal.js.
+     */
+    interface AuthContext {
+        /**
+         * The account unique ID.
```

Comment:
I know I'm an old fogy about stacked nouns, but for some reason this one seems especially annoying. 

```suggestion
         * The unique ID of the account.
```

Reply:
@Rick-Kirkham Yeah, that does sound weird. Fixed!